### ISSUE TITLE
Github login authentication soon deprecated

### ISSUE THREAD

**Issue by Baukebrenninkmeijer** [Toxantron/scrumonline#137]
The user/password based authentication for github will soon be deprecated. Support for the new authentication methods will be required from then on.

**Comment by Toxantron** [comment]
Thanks for the hint. What would your preferred alternative be? Access token?

**Comment by Baukebrenninkmeijer** [comment]
Yeah, either access token or the app access option. The access token way is likely easier, since issue access doesn't require high permissions.
